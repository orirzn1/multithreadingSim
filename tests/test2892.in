L15
S5
O14
N6

T0
I@0x00000000
ADDI $0, $6, 0x4
LOAD $3, $1, $2
STORE $4, $3, 0x2
STORE $3, $6, $2
SUB $2, $2, $2
LOAD $2, $6, $3
LOAD $6, $0, $4
SUBI $2, $0, 0x4
ADD $0, $3, $1
STORE $0, $5, 0x2
HALT $0

T1
I@0x00000000
SUBI $0, $2, 0x1
SUB $4, $3, $5
LOAD $6, $4, $6
STORE $6, $4, $4
STORE $7, $4, 0x1
STORE $6, $3, $5
ADD $7, $4, $7
STORE $4, $3, $1
SUB $3, $3, $1
HALT $0

T2
I@0x00000000
ADD $7, $3, $0
ADD $1, $6, $7
LOAD $0, $2, 0x1
LOAD $5, $7, 0x0
LOAD $3, $1, 0x0
ADD $3, $1, $1
LOAD $0, $7, 0x2
SUB $6, $1, $0
SUB $1, $1, $1
ADDI $2, $4, 0x4
HALT $0

T3
I@0x00000000
STORE $0, $6, $4
ADD $0, $0, $0
SUB $4, $4, $1
SUB $7, $3, $6
SUBI $5, $2, 0x2
LOAD $1, $6, 0x0
SUB $4, $1, $2
HALT $0

T4
I@0x00000000
ADDI $7, $3, 0x2
SUB $5, $7, $3
STORE $3, $7, $7
ADDI $2, $4, 0x4
SUBI $5, $3, 0x2
LOAD $5, $7, 0x4
LOAD $0, $3, 0x4
SUB $0, $7, $4
STORE $7, $0, $1
STORE $1, $7, $6
HALT $0

T5
I@0x00000000
STORE $1, $4, 0x2
LOAD $2, $2, $1
STORE $2, $6, 0x4
STORE $2, $1, 0x1
SUBI $6, $3, 0x3
LOAD $0, $1, $2
HALT $0

D@0x0
0x7
0x3
0x3
0x8
0xA
0xA
0x2
0x4
