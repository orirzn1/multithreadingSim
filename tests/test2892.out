
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x7	R1 = 0x0	R2 = 0x0	R3 = 0x7	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x7	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x7	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x7	R2 = 0x0	R3 = 0x0	R4 = 0x7	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x2	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x2
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x7	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Blocked MT CPI for this program 8.724138

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x2	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x2	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x2	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x2
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Finegrained Multithreading CPI for this program 1.431034

